addons/binding/org.openhab.binding.cm11a/src/main/java/org/openhab/binding/cm11a/handler/Cm11aAbstractHandler.java

```diff
@@ -74,14 +74,18 @@ public Cm11aAbstractHandler(Thing thing) {
 
     @Override
     public void initialize() {
+        final String NO_BRIDGE_ERROR = "No bridge found using house unit code ";
+
         Configuration config = thing.getConfiguration();
-        if (config == null) {
```

Comment:
Why is this not needed, can it not return `null` (anymore)?

Reply:
Eclipse says it can't be null here.